artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumProjectIntegration/Infrastructure/ApiTokens/MockedUserTokensRepository.cs

```diff
@@ -0,0 +1,86 @@
+using System.Net;
+using System.Text.Json;
+using Artificial.Scrum.Master.ScrumProjectIntegration.Exceptions;
+using Artificial.Scrum.Master.ScrumProjectIntegration.Infrastructure.Models;
+using Artificial.Scrum.Master.ScrumProjectIntegration.Settings;
+
+namespace Artificial.Scrum.Master.ScrumProjectIntegration.Infrastructure.ApiTokens
+{
+    public class MockedUserTokensRepository : IUserTokensRepository
+    {
+        private readonly MockupTokenManager _mockupTokenManager;
+
+        public MockedUserTokensRepository(ScrumManagementServiceSettings settings)
+        {
+            _mockupTokenManager = new MockupTokenManager(settings);
+        }
+
+        public async Task<UserTokens?> GetUserAccessTokens(string userId)
```

Comment:
chłopie po co ci taki gigantyczny mock. Przecież to pewnie zajęło więcej niż reszta kodu xD